pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.camel</groupId>
         <artifactId>camel-dependencies</artifactId>
-        <version>3.8.0</version>
+        <version>3.9.0</version>
```

Comment:
@jamesnetherton I've updated this to make it working, I may need to rebase once your PR gets in